spec/policy_loader_spec.lua

```diff
@@ -0,0 +1,24 @@
+local _M = require 'apicast.policy_loader'
+
+describe('APIcast Policy Loader', function()
+
+  describe('.call', function()
+    it('finds apicast builtin policy', function()
+      local root = require('apicast.policy.apicast')
+      local sandbox = _M:call('apicast')
+
+      assert.is_table(sandbox)
+      assert.are_not.same(root, sandbox)
+      assert.equal(root._NAME, sandbox._NAME)
+      assert.equal(root._VERSION, sandbox._VERSION)
```

Comment:
These `asserts` are good, but how can we know that this is really the Apicast policy :eyes: ? Should we at least check that it contains some of its more important methods? The phases for example. I feel like we need something more than testing that it's just a table with the correct name and version. Ideas?

Reply:
The issue is that if you load the same policy twice by the policy loader it is actually different table with different functions. The file is evaluated and returned twice.  And it is always going to be different than the one returned by require.
Each policy has all the phases, so we can't really compare that.

I tried adding there some random id, but it is different every time you load the policy. I really have no nice way of comparing them.